### PR TITLE
compare refdata with arbitrary labels

### PR DESCRIPTION
**Pull request by epassaro**
## Description
Allow pipeline to compare two versions of the reference data by specifying labels (commit hashes, tags, branches, etc.).  

- If no label is specified, the pipeline will run as usual generating new reference data and comparing against the latest version stored in `tardis-refdata` repo.

- Also I found a workaround to fetch LFS data from Azure Mirror with PAT.

**It's working!** See :http://opensupernova.org/~azuredevops/files/refdata-results/1430/d963e15e26a98f56b00778e44a016e547883f26a.html

## Motivation and Context

Part of the CI/CD project.

## How Has This Been Tested?
- [ ] Testing pipeline
- [x] Reference Data Comparison following these [instructions](https://tardis-sn.github.io/tardis/development/continuous_integration.html)
- [ ] Other (please describe)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] None of the above (please describe)

## Checklist:

- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have built the documentation on my fork following [these instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html)
- [x] I have assigned and requested two reviewers for this pull request

### CODE REVIEW

**Comment by epassaro** [comment]
https://tardis-sn@dev.azure.com/tardis-sn/TARDIS/_git/tardis-refdata

File: docs/development/continuous_integration.rst [hunk 1]
```diff
@@ -290,17 +289,28 @@ Release pipeline
...
   /AzurePipelines run compare-refdata
 
 For brevity, you can comment using ``/azp`` instead of ``/AzurePipelines``.
+
+By default, generates new reference data for the ``HEAD`` of the pull request. Then, 
+compares against latest reference data stored in ``tardis-refdata`` repository. The
+web UI also allows to compare any version of the reference data by providing two
+labels (SHAs, branches, tags, etc.) as variables named ``ref1.hash`` and ``ref2.hash``
+at runtime.
+
```
Comment (andrewfullard) [inline docs/development/continuous_integration.rst:308]: This needs more detail

**Review by andrewfullard — CHANGES_REQUESTED** [review]

File: docs/development/continuous_integration.rst (hunk 1 above)
Comment (epassaro) [reply docs/development/continuous_integration.rst:315]: Done

**Review by andrewfullard — APPROVED** [review]

**Pull request merged**